values/cert-manager/cert-manager-raw.gotmpl

```diff
@@ -33,12 +36,17 @@ resources:
           - selector: {}
             dns01:
               {{- index $cm.provider $dnsProvider | toYaml | nindent 14 }}
-                {{- if or (eq $dnsProvider "azure") (eq $dnsProvider "google") }}
+                {{- if eq $dnsProvider "azure" }}
                 clientSecretSecretRef:
-                  name: dns-secret
+                  name: {{ $secretName }}
```

Comment:
please don't make devs think that`$secretName` is configurable...this is our code, we can just hardcode it in these two spots here and remove confusion. Don't think this is making it more DRY

Reply:
you're right